### PR TITLE
Print out namespaces list as json by default

### PR DESCRIPTION
**Pull request by gabbifish**
Make json output default for listing namespaces as well (and update documentation)!

### CODE REVIEW

**Comment by EverlastingBugstopper** [comment]
Do we want to default to pretty print with the table and then print json if we pass a `--json` flag? I feel like that's the usual pattern for tools that output json. Not a hill I'll die on though

**Comment by gabbifish** [comment]
@EverlastingBugstopper I think we can get away without offering pretty-printing right now--there are plenty of other commandline tools one can just pipe the wrangler output to in order to pretty-print it :) (I include an example of this in the docs)

File: docs/content/kv_commands.md [hunk 1]
```diff
@@ -53,16 +53,20 @@ $ wrangler kv:namespace rename f7b02e7fc70443149ac906dd81ec1791 "updated kv name
...
 ```sh
-$ wrangler kv:namespace list
-🌀  Retrieving namespaces
-✨  Success:
-+------------------+----------------------------------+
-| TITLE            | ID                               |
-+------------------+----------------------------------+
-| new kv namespace | f7b02e7fc70443149ac906dd81ec1791 |
-+------------------+----------------------------------+
+$ wrangler kv:namespace list | python -m json.tool
```
Comment (xtuc) [inline docs/content/kv_commands.md]: Why can't we pretty print it from Rust?

Comment (gabbifish) [reply docs/content/kv_commands.md]: Enough people have brought this up that I'll add pretty printing and a flag for it :)

Comment (gabbifish) [reply docs/content/kv_commands.md]: nvm, gonna recommend folks use jq. It's hard for us to pretty-print the keys json since it is streamed.

**Pull request merged**